frontend/src/app/components/connections-list/own-connections/own-connections.component.css

```
@@ -333,16 +334,58 @@
-/* ── Schema button inside card ── */
+/* ── Schema button row at the bottom of the card ── */
 
 .connectionSchemaButton {
-	margin-left: auto;
-	flex-shrink: 0;
-	color: rgba(255, 255, 255, 0.7);
+	position: relative;
+	align-self: stretch;
+	justify-content: center;
+	--mdc-text-button-label-text-color: color-mix(in srgb, var(--color-accentedPalette-500) 18%, #1f2937 82%);
+	color: var(--mdc-text-button-label-text-color) !important;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)